src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,18 +3,46 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private int sizeOfStorage = 0;
+    private K[] keys;
+    private V[] values;
+
+    public StorageImpl() {
+        keys = (K[])new Object[MAX_SIZE];
+        values = (V[])new Object[MAX_SIZE];
+    }
 
     @Override
     public void put(K key, V value) {
+        if (sizeOfStorage >= MAX_SIZE) {
+            throw new RuntimeException("Array is full!");
+        } else {
```

Comment:
```suggestion
        } 
```
we don't need else after exception or return in if